StyleCop.Analyzers/StyleCop.Analyzers.Test/MaintainabilityRules/FileMayOnlyContainTestBase.cs

```diff
@@ -32,18 +33,30 @@ public abstract class FileMayOnlyContainTestBase : CodeFixVerifier
 %1 Bar
 {
 }";
-            var fixedCode = @"%1 Foo
+
+            var fixedCode = new[]
+            {
+                @"%1 Foo
 {
 }
-";
+",
+                @"%1 Bar
+{
+}"
+            };
 
             DiagnosticResult expected = this.CSharpDiagnostic().WithLocation(4, this.Keyword.Length + 2);
 
             await this.VerifyCSharpDiagnosticAsync(testCode.Replace("%1", this.Keyword), expected, CancellationToken.None).ConfigureAwait(false);
-            await this.VerifyCSharpDiagnosticAsync(fixedCode.Replace("%1", this.Keyword), EmptyDiagnosticResults, CancellationToken.None).ConfigureAwait(false);
+
+            foreach (var code in fixedCode)
+            {
+                await this.VerifyCSharpDiagnosticAsync(code.Replace("%1", this.Keyword), EmptyDiagnosticResults, CancellationToken.None).ConfigureAwait(false);
+            }
```

Comment:
:thought_balloon: This code construction is repeated a lot. Might be a good idea to add a VerifyNoCSharpDiagnosticsAsync method that will accept either a source file or a list of sources.